lightning/src/routing/router.rs

```diff
@@ -2111,6 +2111,8 @@ fn add_random_cltv_offset(route: &mut Route, payment_params: &PaymentParameters,
 	let network_nodes = network_graph.nodes();
 
 	for path in route.paths.iter_mut() {
+		if path.blinded_tail.as_ref().map_or(false, |tail| tail.hops.len() > 1) { continue }
```

Comment:
Could probably use a comment as to why this will skip.

Reply:
I think I was concerned we might blow up the cltv expiry if blinded path recipients add their own shadow offsets, but I think it makes more sense to just always add a shadow offset for blinded paths, rather than only adding it for 1-hop blinded paths. Updated.